bin/www

```diff
@@ -1,10 +1,9 @@
-﻿#!/usr/bin/env node
-var debug = require("debug")("OnlineEditorsExampleNodeJS");
-var app = require("../app");
+﻿var debug = console.log;
+var app = require('../app-compiled');

```

Comment:
app-compiled does not exist and is created nowhere - I guess you forgot to commit some kind of build script.

Why would we need to compile anyway? I dont think there is any fancy ES7 stuff that we use, so node v6 should be able to run it all